crates/core/src/lib.rs

```diff
@@ -284,6 +284,11 @@ impl SessionLivedBackend {
         }
     }
 
+    pub fn owner(&self) -> Option<String> {
+        let labels = self.metadata.labels.as_ref()?;
+        labels.get("spawn-owner").map(|l| l.to_string())
```

Comment:
If the type of `l` is `&String` (I'm not 100% but I think it is), the `.map` call could be replaced by `.clone()`. Otherwise, looks good!

Reply:
Updated